Create helper to build public widget options for categorizer (#2092)
Khan/perseus: .changeset/moody-numbers-move.md

---

> "@khanacademy/perseus": patch

Author: I chose patch because I'm just adding content that isn't being used yet, but I am modifying a widget's widgetExport object, so let me know if that means it should be a minor change.

Author: Updated!

---

> Create a function to filter out answer data from categorizer's widget options and add to its widget export

Reviewer: Since this is where we introduce this concept, I'm going to suggest a few small wording improvements. Feel free to edit further. :) 

```suggestion
Introduce a widget export function to filter out scoring data from widget options. Implement this function for the categorizer widget.
```
